doc/_support/devices/default.nix

```diff
@@ -8,7 +8,7 @@
 
 let
   # Release tools used to evaluate the devices metadata.
-  mobileReleaseTools = (import ../../../lib/release-tools.nix);
+  mobileReleaseTools = (import ../../../lib/release-tools.nix { inherit pkgs; });
```

Comment:
Your changes didn't inherit `pkgs` from the docs build, which *could* have introduced a second Nixpkgs.